app/src/ui/index.tsx

```diff
@@ -314,8 +314,7 @@ ipcRenderer.on('focus', async () => {
     selectedState &&
     !(selectedState.type === SelectionType.CloningRepository)
   ) {
-    await dispatcher.refreshTags(selectedState.repository)
-    await dispatcher.refreshRepository(selectedState.repository)
+    dispatcher.refreshRepository(selectedState.repository)
```

Comment:
I've left this method exactly as it was before adding the `refreshTags()` call (which means that it doesn't await `refreshRepository()` to finish before calling `setAppFocusState(true)`).